source/_partial/errors_anatomy.md

```diff
@@ -0,0 +1,9 @@
+1. **Error name**: This is the type of the error (e.g. AssertionError, CypressError)
+2. **Error message**: This generally tells you what went wrong. It can vary in length. Some are short like in the example, while some are long, and may tell you exactly how to fix the error. Some also contain a **Learn more** link that will take you to relevant Cypress documentation.
+3. **View stack trace**: Clicking this toggles the visibility of the stack trace.
+4. **Stack trace**: This is hidden by default, but clicking *View stack trace* will show it in full. Stack traces vary in length. Clicking on the blue file path will open the file in your [preferred file opener](https://on.cypress.io/IDE-integration#File-Opener-Preference).
```

Comment:
Any reason these links are standard markdown links and aren't `{% url $}` tags?

Reply:
😅 Cause hexo is a mess that's why. url tags don't work in partials. https://github.com/cypress-io/cypress-documentation/issues/2422